specifyweb/frontend/js_src/css/main.css

```diff
@@ -348,3 +348,14 @@
 .cm-editor {
   @apply h-full;
 }
+
+.colorPickerClass {
+  position: absolute;
+  clip: rect(0, 0, 0, 0);
```

Comment:
`clip` is not the only property from `sr-only` - sr-only has other important properties too (for compatibility between browsers)
Instead, this can be rewritten as `@apply absolute sr-only`